locale/fa/site.json

```diff
@@ -1,19 +1,19 @@
 {
```

Comment:
Could you pease check if the translations here for `site.json` against those in #1871?